AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -157,7 +159,23 @@ final class InitialSyncOperation: AsynchronousOperation {
         let items = syncQueryResult.items
         recordsReceived += UInt(items.count)
```

Comment:
It seems like we can replace `recordsReceived` with `modelSyncedPayload` or a calculated property thereof.